lib/read_table_A.pl

```diff
@@ -2077,18 +2124,16 @@ sub read_table_A {
                 $loc .= ',1,1' if ( $loc =~ tr/,/,/ ) < 3;
                 $code .= sprintf "\$%-35s -> set_fp_location($loc);\n", $name;
             }
-            $code .= sprintf "\$%-35s =  new Group;\n", $group
-              unless $groups{$group};
-            $code .= sprintf "\$%-35s -> add(\$%s);\n", $group, $name
-              unless $groups{$group}{$name};
-            $groups{$group}{$name}++;
+            unless ($groups{$group}) {
+                $code .= sprintf "\$%-35s =  new Group unless (\$%s);\n", $group, $group;
+            }
+            unless ($groups{$group}{$name}) {
+                $code .= sprintf "\$%-35s -> add(\$%s);\n", $group, $name;
+                $groups{$group}{$name}++;
+            }
         }
     }
 
-    if ($additional_code) {
-        $code .= $additional_code;
-    }
-
```

Comment:
I didn't read the whole diff carefully, but I believe you may be running this code at a different time now, and might generate code that has a different order?

Reply:
Thanks. Will cancel this PR if I have the authority. I'll resubmit, regardless.